### DISCUSSION TITLE
Allow pen buttons to be disabled within tablet settings

### DISCUSSION THREAD

**Discussion by Airiuwu** [ppy/osu#14036] [Ideas] [answered]
I personally have a broken pen without a nib or buttons and it thinks I am clicking 24/7

**Comment by frenzibyte** [comment, marked as answer]
Pen buttons by default are not bound to anything, if you have it bound then you can easily unbind that.

As for the nib/tip, that'll be resolved with a "tip button" and "tip pressure" setting you can configure to stop that. Tracking at #14037
